lib/tag-finder.coffee

```diff
@@ -8,8 +8,8 @@ SelfClosingTags = require './self-closing-tags'
 module.exports =
 class TagFinder
   constructor: (@editor) ->
-    @tagPattern = /(<(\/?))([^\s>]+)([^/>]*)(\/?)(>)/
-    @wordRegex = /[^>\r\n]*>/
+    @tagPattern = /(<(\/?))([^/\s>]+)([^/>]*)(\/?)(>)/
+    @wordRegex = /(?![\n\r])[^>]*>/
```

Comment:
Two things:
1. Can the `(?![\n\r])` be changed to simply be `(?!$)`?
2. Either way, please add the lookahead at the end of the match.